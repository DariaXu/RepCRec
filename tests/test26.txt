// Test that sites can go down before they are written.
begin(T1)
R(T1, x1)
fail(2)
recover(2)
W(T1, x1, 22)
end(T1)
// ---
// assertCommitted(T1)
