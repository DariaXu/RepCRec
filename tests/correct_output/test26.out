T1 reads - x1: 10
Abort: T1
